### PR TITLE
Serving files

### PR DESCRIPTION
**Pull request by andrecardoso**

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(8) Create HTTP server](https://trello.com/c/OsIFhO9u/4090-8-create-http-server)

**Comment by andrecardoso** [comment]
[(3) Handle GET requests when file is not found](https://trello.com/c/70WXrzEM/4093-3-handle-get-requests-when-file-is-not-found)

**Comment by andrecardoso** [comment]
[(2) Start and shutdown HTTP server](https://trello.com/c/Zzp1MvxJ/4094-2-start-and-shutdown-http-server)

File: src/storage/file-server.js [hunk 1]
```diff
@@ -95,10 +99,33 @@ function sendResponse(socketId, buffer, keepAlive) {
...
+    `HTTP/1.0 ${httpStatus}`,
+    `Content-Length: ${contentLength}`,
+    `Content-Type: ${contentType}`
+  ];
+
+  if (keepAlive) {
+    lines.push('Connection: keep-alive');
+  }
+
+  const responseText = lines.join('\n') + '\n\n'; // eslint-disable-line
```
Comment (tejohnso) [inline src/storage/file-server.js]: If I recall correctly, the spec is '\r\n' - did you try that?

Comment (tejohnso) [reply src/storage/file-server.js]: Oh...Yeah, we should go with \r\n unless it's a problem.
https://stackoverflow.com/questions/5757290/http-header-line-break-style

Comment (andrecardoso) [reply src/storage/file-server.js]: Yes, I tried and it works as is. I got it from here: https://github.com/GoogleChrome/chrome-app-samples/blob/master/samples/webserver/index.js#L105 

But you are completely right. I'm going to change it to respect the protocol. 

Thanks!

**Pull request merged**